packages/@okta/vuepress-theme-prose/layouts/Layout.vue

```diff
@@ -5,19 +5,19 @@
     </div>
 
     <div class="page-body">
-      <Breadcrumb />
+      <Breadcrumb/>
```

Comment:
To keep with consistency, lets add the space back in here. Typically, self closing tags in Vue have `<Component />`